src/Compilers/CSharp/Portable/Symbols/NamespaceOrTypeSymbol.cs

```diff
@@ -324,7 +324,7 @@ internal virtual NamedTypeSymbol LookupMetadataType(ref MetadataTypeName emitted
                 }
             }
 
-        Done:
+Done:
```

Comment:
i believe the former is a supported label position.  perhaps the compiler/Roslyn should be setting that formatting option in editor config?